ansible/vagrant/README.md

```diff
@@ -1,14 +1,52 @@
-## Vagrant deployer for Kubernetes Ansible
+## Welcome to the Vagrant Deployer for Kubernetes Ansible
 
-This deployer sets up a vagrant cluster and installs kubernetes with flannel on it.
+This deployer sets-up a Kubernetes cluster on Vagrant.
 
-## Before you start !
+## Before You Start
 
-You will need a functioning vagrant provider. Currently supported are openstack, libvirt, and virtualbox.
+Make sure you have [git installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and clone the contrib repo:
+```
+git clone https://github.com/kubernetes/contrib.git
+```
+
+[Install Vagrant](https://www.vagrantup.com/downloads.html) if it's not currently installed on your system.
+
+You will need a functioning [vagrant provider](https://www.vagrantup.com/docs/providers/). Currently supported providers are openstack, libvirt, and virtualbox. Vagrant comes with VirtualBox support by default. No matter what provider you choose, you need to install the OpenStack Vagrant plugin:
```

Comment:
It was definitely required in my testing even though I was using vbox. I believe due to: https://review.openstack.org/#/c/241866/